package.json

```diff
@@ -7,6 +7,7 @@
   },
   "scripts": {
     "lint": "eslint .",
+    "pretest": "node test/helpers/pretest.js",
```

Comment:
Try this: https://github.com/cucumber/cucumber-js/issues/261#issuecomment-59885494 for beforeAll/afterAll hooks. I don't like how its not documented well at all yet.